prow/jobs/jinja/presubmits/code_generator_tests.jinja2

```diff
@@ -38,6 +38,8 @@
           value: "s3"
         - name: ACK_GENERATE_OLM
           value: "true"
+        - name: OPERATOR_SDK_BIN_PATH
+          value: "/usr/local/bin"
```

Comment:
I think I would prefer for this to be maintained inside the image. Since we are installing the binary as part of the Dockerfile, if we update that, we would also need to remember to update this Prow config. Would you mind just adding the export on a line underneath where we copy the operator SDK binary over? https://github.com/aws-controllers-k8s/test-infra/blob/main/prow/jobs/images/Dockerfile.olm-bundle-pr#L53-L54

Reply:
@RedbackThomson I moved this change to the docker files, and also made these two docker files match where appropriate. LMK if there are any other issues.

Reply:
I am not sure if exporting the variable in a Dockerfile layer will work make the env variable available during prow execution.


Why not add `ENV OPERATOR_SDK_BIN_PATH=/usr/local/bin` inside both Dockerfiles?

Reply:
@vijtrip2 I wasn't sure either, that's why I added it to the `jobs` first, but you're probably right then need to be ENV's as well.